src/clients/utils.ts

```diff
@@ -36,3 +36,11 @@ export function trimLastForwardSlash(url: string) {
 export function capitalizeFirstLetter(string: string) {
   return string.length > 0 ? `${string[0].toUpperCase()}${string.slice(1)}` : ''
 }
+
+export function handleUrlFilters<T>(filterKey: string, value: T | null, params: URLSearchParams) {
```

Comment:
Love this! Maybe switch `value` and `params` so value can be optional and we don't need to be calling the function with `null`?